src/io/flutter/logging/FlutterConsoleLogManager.java

```diff
@@ -144,4 +172,54 @@ private void printStackTraceToConsole(
   private String stringValueFromStringRef(InstanceRef ref) {
     return ref.getValueAsStringIsTruncated() ? ref.getValueAsString() + "..." : ref.getValueAsString();
   }
+
+  private String stringValueFromStringRef(Instance instance) {
+    return instance.getValueAsStringIsTruncated() ? instance.getValueAsString() + "..." : instance.getValueAsString();
+  }
+
+  private String getFullStringValue(@NotNull VmService service, String isolateId, @Nullable InstanceRef ref) {
```

Comment:
Would it make sense to add a couple unit tests given the different cases for what gets written to the `GetObjectConsumer`?

Reply:
👍 yes, good point. I’ll see about where these can slot into our existing tests